Configurations/10-main.conf

```diff
@@ -748,6 +750,21 @@ sub vms_info {
         perlasm_scheme   => "a.out",
     },
 
+    #### X86 / X86_64 targets
+    "linux-x86" => {
+        inherit_from     => [ "linux-generic32", asm("x86_asm") ],
+        cflags           => add(picker(default => "-m32 -DL_ENDIAN",
+                                       release => "-fomit-frame-pointer")),
+        bn_ops           => "BN_LLONG",
+        perlasm_scheme   => "elf",
```

Comment:
Right. I was assuming that you would inherit x86_elf_asm, as linux-elf does. Well, personally I'd prefer to see x86_elf_asm, but it doesn't matter much...